.github/workflows/run-bdd-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Jest tests
+on:
+  push:
+    branches:
+      - feat/manager-shell
```

Comment:
NB: Code to avoid pollution on other pull requests. This restriction will be removed after approvals